src/main/java/com/virtudoc/web/controller/Notification.java

```diff
@@ -0,0 +1,63 @@
+package com.virtudoc.web.controller;
+import java.time.LocalDateTime;
+
+public class Notification {
```

Comment:
If you were wanting this to be stored in the database, you'll need to add `@Entity` on line 3. You'll also need a `private Integer id` field with the `@Id` and `@GeneratedValue(strategy= GenerationType.AUTO)` attributes.